_html/scss/layout/_grid.scss

```diff
@@ -0,0 +1,34 @@
+.l-row {
+    max-width: 64rem;
+
+    &.l-column {
+        margin: 0 auto;
+    }
+
+    .l-row {
+        margin: 0 .625rem;
+
+        @include tablet {
+            margin: 0 .9375rem;
+        }
+    }
+}
+
+.l-columns {
+    width:         100%;
+    padding-right: .625rem;
+    padding-left:  .625rem;
```

Comment:
Please use one line declaration for padding.

Reply:
Done.